src/utils/services/axios.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import AuthService from '../../Services/AuthService';
+export const axiosInstance = axios.create({
+  baseURL: process.env.REACT_APP_BACKEND_URL,
+});
+const accessToken = AuthService.getToken();
+export const axiosGet = async (url, params = {}, contentType = 'application/json') => {
+  let response = {};
+  try {
+    const result = await axiosInstance.get(url, {
+      headers: {
+        'Content-Type': contentType,
+        'Authorization': `Bearer ${accessToken}`,
+      },
+      params,
+    });
+    response = result.data || {};
+    response.status = result?.status === 200;
```

Comment:
Successful result statuses can be other than 200, let's not hardcore the statuses to 200. (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses)